packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/AddressItem.tsx

```diff
@@ -0,0 +1,84 @@
+import { WalletId } from '@hicommonwealth/shared';
+import { formatAddressShort } from 'client/scripts/helpers';
+import AddressInfo from 'client/scripts/models/AddressInfo';
+import NewProfile from 'client/scripts/models/NewProfile';
+import {
+  handleMouseEnter,
+  handleMouseLeave,
+} from 'client/scripts/views/menus/utils';
+import useAuthentication from 'client/scripts/views/modals/AuthModal/useAuthentication';
+import React from 'react';
+import { PopoverMenu } from '../../component_kit/CWPopoverMenu';
+import { CWIconButton } from '../../component_kit/cw_icon_button';
+import { CWIcon } from '../../component_kit/cw_icons/cw_icon';
+import { CWText } from '../../component_kit/cw_text';
+import { CWTooltip } from '../../component_kit/new_designs/CWTooltip';
+
+import './AddressItem.scss';
+
+type AddressItemProps = {
+  profile: NewProfile;
+  addressInfo: AddressInfo;
+  toggleRemoveModal: (val: boolean, address: AddressInfo) => void;
+  isSelected: boolean;
+};
+
+const AddressItem = (props: AddressItemProps) => {
+  const { addressInfo, toggleRemoveModal, isSelected } = props;
+  const { address, walletId } = addressInfo;
+
+  const { openMagicWallet } = useAuthentication({});
+
+  return (
+    <div className="AddressItem">
+      <div className="address">
+        <CWText className="address-label" type="b2" fontWeight="regular">
+          {formatAddressShort(address, 6)}
+        </CWText>
+      </div>
+      {isSelected && <CWIcon iconName="checkCircleFilled" />}
```

Comment:
desgin vs current, adjust size and coolor of icon
![image](https://github.com/user-attachments/assets/71c671ec-c1b8-42fa-95a4-834d34b47893)
![image](https://github.com/user-attachments/assets/2e7cae7c-b17f-466f-8422-b5e06641e895)